src/Rings/mpoly-ideals.jl

```diff
@@ -2072,8 +2085,36 @@ function flag_pluecker_ideal(F::Field, dimensions::Vector{Int}, n::Int)
   genz = [sum(t) for t in T]
   filter!(!iszero, genz)
   gg = unique(x ->  divexact(x, canonical_unit(x)), genz)
+  I  = ideal(gg)
 
-  return ideal(gg)
+  !minimal && return I 
+  # Computes the groebner_basis of the ideal
+  converted_generators = collect(groebner_basis(I; ordering = degrevlex(base_ring(I))))
+  o = degrevlex(R)
+  return ideal(IdealGens(R, converted_generators, o;
+                   keep_ordering=true,
+                   isReduced=true,
+                   isGB=true))
 end
 
-flag_pluecker_ideal(dimensions::Vector{Int}, n::Int) = flag_pluecker_ideal(QQ,dimensions,n)
+flag_pluecker_ideal(dimensions::Vector{Int}, n::Int; minimal::Bool=true) = flag_pluecker_ideal(QQ,dimensions,n, minimal=minimal)
+
+function flag_pluecker_ideal(ring::MPolyRing, dimensions::Vector{Int}, n::Int; minimal::Bool=true)
```

Comment:
since you are expecting the coefficients to belong to a field you should add that to the type signature
```suggestion
function flag_pluecker_ideal(ring::MPolyRing{<: FieldElem}, dimensions::Vector{Int}, n::Int; minimal::Bool=true)
```